mLRS/Common/setup.h

```diff
@@ -396,6 +411,20 @@ void setup_configure(void)
     case SETUP_FREQUENCY_BAND_868_MHZ:
```

Comment:
BUG BUG BUG
`break;` is missing here !!!!

(this is also in master, and I wonder how 915 MHZ FCC ever could work !!)

Reply:
Isn't break for 868 just on the next line? 

![image](https://user-images.githubusercontent.com/41841496/222286870-e6e7e6f0-223b-4fb1-8683-89fcf0026d51.png)

Reply:
Oh I see before this - my bad.